package sagutdinov.exception

class UserExistsException(message: String): RuntimeException(message)